.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thank you for helping to improve Upbound!
+
+Please read through https://git.io/fj2m9 if this is your first time opening an
+Upbound pull request. Find us in https://slack.crossplane.io/messages/upbound if
+you need any help contributing.
+-->
+
+### Description of your changes
+
+<!--
+Briefly describe what this pull request does. Be sure to direct your reviewers'
+attention to anything that needs special consideration.
+
+We love pull requests that resolve an open issue. If yours does, use the below
+line to indicate which issue your PR fixes, for example "Fixes #500":
+-->
+
+Fixes #
+
+I have:
+
+- [ ] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
+- [ ] Run `make reviewable test` to ensure this PR is ready for review.
```

Comment:
Relatively few of our repositories support the `make reviewable` command; I think it's time we add it to the upbound/build repo as a standard command.